workflows/dnaseq/dnaseq-standard-fastq.wdl

```diff
@@ -0,0 +1,114 @@
+            fields: {
+                ID: "Read group identifier. Each Read Group must have a unique ID. The value of ID is used in the RG tags of alignment records.",
+                BC: "Barcode sequence identifying the sample or library. This value is the expected barcode bases as read by the sequencing machine in the absence of errors. If there are several barcodes for the sample/library (e.g., one on each end of the template), the recommended implementation concatenates all the barcodes separating them with hyphens (`-`).",
+                CN: "Name of sequencing center producing the read.",
+                DS: "Description.",
+                DT: "Date the run was produced (ISO8601 date or date/time).",
+                FO: "Flow order. The array of nucleotide bases that correspond to the nucleotides used for each flow of each read. Multi-base flows are encoded in IUPAC format, and non-nucleotide flows by various other characters. Format: /\\*|[ACMGRSVTWYHKDBN]+/",
+                KS: "The array of nucleotide bases that correspond to the key sequence of each read.",
+                LB: "Library.",
+                PG: "Programs used for processing the read group.",
+                PI: "Predicted median insert size, rounded to the nearest integer.",
+                PL: "Platform/technology used to produce the reads. Valid values: CAPILLARY, DNBSEQ (MGI/BGI), ELEMENT, HELICOS, ILLUMINA, IONTORRENT, LS454, ONT (Oxford Nanopore), PACBIO (Pacific Biosciences), SINGULAR, SOLID, and ULTIMA. This field should be omitted when the technology is not in this list (though the PM field may still be present in this case) or is unknown.",
+                PM: "Platform model. Free-form text providing further details of the platform/technology used.",
+                PU: "Platform unit (e.g., flowcell-barcode.lane for Illumina or slide for SOLiD). Unique identifier.",
+                SM: "Sample. Use pool name where a pool is being sequenced."
+            }
+        }
+        prefix: "Prefix for the BAM file. The extension `.bam` will be added."
+        aligner: {
+            description: "BWA aligner to use",
+            choices: ["mem", "aln"]
+        }
+        validate_input: "Ensure input BAM is well-formed before beginning harmonization?"
+        use_all_cores: "Use all cores? Recommended for cloud environments."
+        subsample_n_reads: "Only process a random sampling of `n` reads. Any `n`<=`0` for processing entire input."
+    }
+    input {
+        Array[File] read_one_fastqs_gz
+        Array[File] read_two_fastqs_gz
+        File bwa_db
+        Int reads_per_file = 10000000
+        Array[ReadGroup] read_groups
+        String prefix
+        String aligner = "mem"
+        Boolean validate_input = true
+        Boolean use_all_cores = false
+        Int subsample_n_reads = -1
+    }
+
+    call dnaseq_core_wf.parse_input { input:
+        aligner
```

Comment:
What happens if `len(read_ones) != len(read_twos) != len(read_groups)`? That check probably belongs in `parse_input`.